test/e2e/kubectl/kubectl.go

```diff
@@ -1071,7 +1071,18 @@ metadata:
 			framework.ExpectNotEqual(schema, nil, "retrieving a schema for the crd")
 
 			meta := fmt.Sprintf(metaPattern, crd.Crd.Spec.Names.Kind, crd.Crd.Spec.Group, crd.Crd.Spec.Versions[0].Name, "test-cr")
-			validArbitraryCR := fmt.Sprintf(`{%s,"spec":{"bars":[{"name":"test-bar"}],"extraProperty":"arbitrary-value"}}`, meta)
```

Comment:
the change to `validArbitraryCR` should be in the `Adjust tests to pass for both client-side and server-side validation` commit (and be picked to 1.23)

Reply:
the addition of `invalidArbitraryCR` is good to remain in this commit

Reply:
done